applications/ContactStructuralMechanicsApplication/custom_utilities/derivatives_utilities.cpp

```diff
@@ -662,13 +662,19 @@ void DerivativesUtilities<TDim, TNumNodes, TFrictional, TNormalVariation, TNumNo
                 double DeltaXi1 = 0.0, DeltaXi2 = 0.0;
 
                 for (IndexType i_mortar_node = 0; i_mortar_node < TNumNodes; ++i_mortar_node) {
-                    // Compute DeltaXi on the slave side if point is inside geometry (nodes of geometry excluded)
-                    if (rSlaveGeometry.IsInside( projected_in_slave[i_mortar_node].Coordinates(), aux_point_slave.Coordinates() ) && (norm_2(projected_in_slave[i_mortar_node].Coordinates() - rSlaveGeometry[0].Coordinates()) > ZeroTolerance) && (norm_2(projected_in_slave[i_mortar_node].Coordinates() - rSlaveGeometry[1].Coordinates()) > ZeroTolerance))
+                    // Compute DeltaXi on the slave side if point is inside geometry (nodes off geometry excluded)
```

Comment:
This is the actual error

Reply:
so you've removed this line? :D Which was the problem in simple words?